README.md

```diff
@@ -37,6 +37,18 @@ echo $packages[0]->getVersion();
 echo $packages[0]->getNamespace();
 ```
 
+Getting just production packages.
+```php
+$composerInfo = new ComposerLockParser\ComposerInfo('/path/to/composer.lock');
+$packages = $composerInfo->getPackages(1);
+```
+
+Getting just development packages.
+```php
+$composerInfo = new ComposerLockParser\ComposerInfo('/path/to/composer.lock');
+$packages = $composerInfo->getPackages(1);
```

Comment:
there should be 0?
I think you can use a constant here, something like ComposerInfo::DEVELOPMENT

Reply:
The last one should be a 2, not a 1. I'll submit a commit for it. 

I mean I guess it could be constants, so long as it can be passed to the function.

Reply:
Submitted change